internal/namespaces/instance/v1/updated_commands.go

```diff
@@ -46,3 +48,81 @@ func updateInstancePlacementGroupGet(c *core.Command) {
 		},
 	}
 }
+
+func updateInstanceServerUpdate(c *core.Command) {
+
+	type instanceUpdateServerRequestCustom struct {
+		*instance.UpdateServerRequest
+		IP *instance.NullableStringValue
+	}
+
+	type instanceUpdateServerResponseCustom struct {
+		*instance.UpdateServerResponse
+		*instance.UpdateIPResponse
+	}
+
+	IPArgSpec := &core.ArgSpec{
+		Name:  "ip",
+		Short: `IP that should be attached to the server (use ip=none to remove)`,
+	}
+
+	c.ArgsType = reflect.TypeOf(instanceUpdateServerRequestCustom{})
+
+	c.ArgSpecs = append(c.ArgSpecs, IPArgSpec)
+
+	c.Run = func(ctx context.Context, argsI interface{}) (i interface{}, e error) {
+
+		customRequest := argsI.(*instanceUpdateServerRequestCustom)
+
+		updateServerRequest := customRequest.UpdateServerRequest
+
+		updateIPRequest := (*instance.UpdateIPRequest)(nil)
+
+		switch {
+		case customRequest.IP == nil:
+			// ip is not set
+			// do nothing
+
+		case customRequest.IP.Null:
+			// ip=none
+			// remove server from ip
+			updateIPRequest = &instance.UpdateIPRequest{
+				IP: customRequest.IP.Value,
+				Server: &instance.NullableStringValue{
+					Null: true,
+				},
+			}
+
+		default:
+			// ip=<anything>
+			// update ip
+			updateIPRequest = &instance.UpdateIPRequest{
+				IP: customRequest.IP.Value,
+				Server: &instance.NullableStringValue{
+					Value: customRequest.ServerID,
+				},
+			}
+		}
+
+		client := core.ExtractClient(ctx)
+		api := instance.NewAPI(client)
+
+		updateServerResponse, err := api.UpdateServer(updateServerRequest)
+		if err != nil {
+			return "", err
+		}
+
+		updateIPResponse := (*instance.UpdateIPResponse)(nil)
```

Comment:
Let's update IP first. This way the correct ip will be present inside the UpdateServer response.